File: Python/RunsOnMyPi/InClassSsh/blink.py
import gpiozero as gz
import time

def main():
    print("Ready to blink via main")
    my_led = gz.LED(18)

    while True:
        my_led.on()
        # print("The LED should be on")
        time.sleep(1)
        my_led.off()
        # print("The LED should be off")
        time.sleep(1)



main()

File: Node/discoveringAPIsTuesday/node_led_blink/blink_template/blink.js
// Imports

function msleep(n) {
    Atomics.wait(new Int32Array(new SharedArrayBuffer(4)), 0, 0, n);
 }
 
 function sleep(n) {
    msleep(n * 1000);
 }

function main() {
    console.log("Ready");

    // TODO: Set GPIO 14 as an output

    while (true) {
        console.log("LED On");
        // TODO: Turn the LED on using a library
        sleep(1);

        console.log("LED Off");
        // TODO: Turn the LED off using a library
        sleep(1);
    }
}


main();
